src/shuffle.js

```diff
@@ -337,7 +337,7 @@ class Shuffle extends TinyEmitter {
   _getItems() {
     return Array.from(this.element.children)
       .filter((el) => matches(el, this.options.itemSelector))
-      .map((el) => new ShuffleItem(el));
+      .map((el) => new ShuffleItem(el, this.options));
```

Comment:
The constructor for `ShuffleItem` expects a boolean.

```suggestion
      .map((el) => new ShuffleItem(el, this.options.isRTL));
```

Reply:
Yea you right, I just pass the whole options to constructor because maybe we need any other options.